components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LoginResource.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.integrator.management.apis;
+
+import com.nimbusds.jose.JOSEException;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseConstants;
+import org.apache.synapse.config.SynapseConfiguration;
+import org.json.JSONObject;
+import org.wso2.micro.integrator.management.apis.security.handler.AuthConstants;
+import org.wso2.micro.integrator.management.apis.security.handler.JWTConfig;
+import org.wso2.micro.integrator.management.apis.security.handler.JWTInMemoryTokenStore;
+import org.wso2.micro.integrator.management.apis.security.handler.JWTTokenGenerator;
+import org.wso2.micro.integrator.management.apis.security.handler.JWTTokenInfoDTO;
+import org.wso2.micro.integrator.management.apis.security.handler.JWTTokenStore;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public class LoginResource implements MiApiResource {
```

Comment:
Class description

Reply:
Thanks fixed in a07cc9b1dc0d33b5dfc4e0af9e81f4611021cb22